sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1673,6 +1673,19 @@ object SQLConf {
       .checkValues(Dialect.values.map(_.toString))
       .createWithDefault(Dialect.SPARK.toString)
 
+  val ANSI_ENABLED = buildConf("spark.sql.ansi.enabled")
+    .doc("This configuration will be deprecated in the future releases and replaced by" +
```

Comment:
`is deprecated and will be removed in future releases`

Reply:
Thanks, done in 1cedb3d.